src/dispensary.js

```diff
@@ -261,7 +263,11 @@ export default class Dispensary {
         hashes.add(hash);
       }
 
-      resolve(Array.from(hashes));
+      var hashesArray = Array.from(hashes).sort((a, b) => {
+        return natCompare(a.match(/\s+(.*)/)[1], b.match(/\s+(.*)/)[1]);
```

Comment:
`a` and `b` look like
`<SHA256 hash> <filename>`

So, the regex finds the filename and sorts by that.

I tried to find better ways to do this but `Set`s are not sortable and we generate this checksum-filename string quite early.

I can add a comment to make this clear.